rom/laghos_utils.cpp

```diff
@@ -354,6 +355,8 @@ void SetWindowParameters(Array2D<int> const& twparam, ROM_Options & romOptions)
     romOptions.dimE = min(romOptions.max_dimE, twparam(w,2));
     romOptions.dimFv = romOptions.SNS ? romOptions.dimV : min(romOptions.max_dimFv, twparam(w,3));
     romOptions.dimFe = romOptions.SNS ? romOptions.dimE : min(romOptions.max_dimFe, twparam(w,4));
+    if (romOptions.useXV) romOptions.dimX = romOptions.dimV;
```

Comment:
The `useXV` and `useVX` options have never been used, and SNS makes them even more obsolete. We probably should remove these options, not necessarily in this PR.